### ISSUE TITLE
Crowdsourced drop rates

### ISSUE THREAD

**Issue by kalilistic** [goatcorp/suggestions#509]
### Description
See title.

### More info
Maybe a fit for Kapture but the bigger effort here be running a server for this.
https://discord.com/channels/581875019861328007/685275026156683280/861843886997241867

----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Dalamud's [plugin guidelines](https://github.com/goatcorp/FFXIVQuickLauncher#isnt-this-cheating)

**Comment by johnysandels** [comment]
that would be so handy! It's so rough to not know the drop rates of anything. aside from dungeon and trial drops, The lockboxes in eureka and Bozja would definitely be something I'd like to be be part of this.

**Comment by rrbailey89** [comment]
yes please, this would be super awesome. maybe the database could be decentralized to save money... although there would still be the need for a server I think.. but idk how these things work, could it be made peer to peer?

**Comment by kalilistic** [comment]
I've considered doing this before... I think would need some thought on what data to collect. Would the content runs (e.g. dungeons/trials) and dropped items just be it?

E.g. Ran ex trial 10 times and 1 mount drop.

**Comment by lmcintyre** [comment]
I am always weary of crowdsourcing data, as people are horrible and will taint it in any way they can, simply because they want to. I guess that hasn't happened with Universalis yet, but it depends on the difficulty of spoofing the data. Just an annoying thing to think about.

**Comment by Eternita-S** [comment]
some things to consider:
- need somehow handle multiple people with plugin in the same group, like, if 4 people in party have plugin installed - need not to end up with 4 records of "ran ex trial 10 times and 1 mount dropped"
- need somehow handle and exclude runs with unopened chests. Let's say someone ran a dungeon and opened only 3 out of 5 chests, drop stats would be invalid in this case since more items could have been in unopened chest. Or someone joined party in progress, they couldn't see previous loot and counting them will generate incorrect data
